cd/application-deployment/dev/vaec-api-task-definition.json

```diff
@@ -327,8 +327,8 @@
   "requiresCompatibilities": [
     "FARGATE"
   ],
-  "cpu": "512",
-  "memory": "1024",
+  "cpu": "2048",
+  "memory": "4096",
```

Comment:
will we need to do this for the other environments?

Reply:
I don't know.  @k-macmillan ???

Reply:
We have not ever tried to deploy this to staging, which I know has a smaller config than perf and prod (like dev was).

Reply:
We should see how dev performs at this and make changes to the other envs as necessary. I picked 2 because I wanted to ensure it wasn't a problem, we probably only need 1.